std_msgs/Bool data
---
geometry_msgs/Pose pose
int8 side
